CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-This project is Open Source. That means that you can contribute on enhancements, bug fixing or improving the documentation in the [Wiki](https://github.com/docker-mailserver/docker-mailserver/wiki).
+This project is Open Source. That means that you can contribute on enhancements, bug fixing or improving the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge).
```

Comment:
Does the `edge` here mean that the source of the pages would be a branch called `edge`? Wasn't it only the tag on the Docker Hub that is called edge? The branch that this pr points to however is still master.

Reply:
`edge` just reflects the state of the docs on `master` branch. It's equivalent to `master`/`latest`, which is what `edge` tagged docker images are meant to reflect AFAIK?

It can be whatever label maintainers would prefer, I assume `edge` was chosen to match the docker `edge` tag since the project itself is docker focused and the docs are for those releases? When a new version is tagged, it'll build the docs from their state in the tagged commit and that will be selectable from a drop-down menu, allowing us to retain old docs as breaking changes and deprecations are introduced.